src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -78,7 +81,11 @@ class MovingWindow:
     Example: Create a polars data frame from a `MovingWindow`
 
         ```python
+        # pylint: disable=import-error
         import polars as pl
```

Comment:
If we remove polars we can remove the whole example. So the discussion is more about do we need the example and my feeling is, that removing an example to make a tool work is not the best practice.

Reply:
OK, I see, although the discussion is not if we need the example or not. I think we all agree we want the example, I just thought maybe it was possible to do some example/demonstration just using `numpy` that was equivalent (since we already have it as a dependency). This is just exposing my ignorance about pandas/numpy if I'm making nonsense, but I didn't wanted to remove examples to adapt to the tool :)

In any case I think we need to strike a balance, because also adding obscure `# pylint: disable=import-error` (that could be carried out blinding by users) is also not great. 

Adding support to specify dependencies in code examples is also quite a bit of work and will make running this examples much slower. An alternative could be to *offload* examples needing external libraries to `examples/`, where it should be easier to specify dependencies for example, but I agree in this case that defeats the whole purpose of having a quick small example embedded.

In any case, I agree that for now if there is no consensus about this, the best approach is to leave the `# pylint: disable=import-error`, but I think the goal should be to somehow remove it in the future. I will create a discussion about this so it's not forgotten.